### DISCUSSION TITLE
Collapse折叠面板在点击展开关闭的时候会触发路由？

### DISCUSSION THREAD

**Discussion by peter-dou** [element-plus/element-plus#14559] [Q&A]
Collapse折叠面板3.0.10没有这种问题 但是3.0.14点击展开关闭会触发路由？

**Comment by jinsonsongg** [comment]
更新版本看看，之前有同样问题，我更新最新就没复现了

**Comment by trcat** [comment]
我也遇到了同样的问题，collapse 折叠面板展开和收起的部分现在变成了按钮，且没有默认设定 `type="button"`，如果这是在一个表单的上下文环境里就会出现路由跳转的问题。

因为我们有 fork element plus，所以是直接修改内部逻辑～

**Reply by jinsonsongg** [reply]
测试还发现在折叠面板的表单里按回车也有概率出现跳转

**Reply by trcat** [reply]
大概是同样的问题，你可以通过浏览器控制台手动给控制折叠展开的按钮添加 button type 验证
